snap/snapcraft.yaml

```diff
@@ -7,16 +7,19 @@ grade: devel
 base: core18
 
 apps:
-  mir-kiosk-app-daemon:
+  daemon:
     command: bin/mir-kiosk-app-daemon
     daemon: simple
-    plugs:
-      - network
-      - opengl
-      - wayland
+    restart-condition: on-failure
+
+  mir-kiosk-apps:
+    command: bin/mir-kiosk-apps
+plugs:
+  network:
+  opengl:
+  wayland:
```

Comment:
Shouldn't these be 
```
 - network
 - opengl
 - wayland
```

Reply:
No. This really is the format in a top level `plugs`, you're confusing it with `apps.<name>.plugs` which is still the way you write.

Reply:
Indeed, it is a dict here. Ref: https://snapcraft.io/docs/snapcraft-yaml-reference